addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14039,7 +14039,7 @@ msgstr ""
 
 #: system/peripherals.xml
 msgctxt "#36033"
-msgid "Pause playback when switching to another source"
+msgid "Action on switching to another source"
```

Comment:
While changing this string, would you mind adding a contextual help string that indicates that this is CEC related and/or gives more background info for translators?

And I'm no native speaker, but `Action when switching to another source` might be better? Or could we get that even shorter? Reason for shorter labels would be that the peripheral dialog is quite narrow and labels tend to scroll or worse (like overlap with the values so that you can't read anything) @jjd-uk ?